katalog/node-exporter/kustomization.yaml

```diff
@@ -1,5 +1,18 @@
+# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
 namespace: monitoring
 
+bases:
+  - dashboards
+
+images:
+  - name: quay.io/prometheus/node-exporter
+    newTag: v1.0.1
+  - name: quay.io/coreos/kube-rbac-proxy
+    newTag: v0.5.0
```

Comment:
Can we use the same version as in `kube-proxy-exporter`?

Reply:
I'll take a look but we are not using the same image.